roofit/roofitcore/src/RooFactoryWSTool.cxx

```diff
@@ -228,7 +228,84 @@ RooCategory* RooFactoryWSTool::createCategory(const char* name, const char* stat
   return _ws->cat(name) ;
 }
 
+bool isEnum(const char* classname) {
```

Comment:
If those routines are only for use inside this file they should be either prefix with R__ or placed in an anonymous namespace.  Either way, they also need to be declared static.  thanks.